### ISSUE TITLE
Does cassandra and mosdef cassandra link back to mosdef.org anywhere?

### ISSUE THREAD

**Issue by justinGilmer** [mosdef-hub/mosdef-hub.github.io#23]
How about any other group and their engines/products?

**Comment by rsdefever** [comment]
Yes! We link back to [mosdef.org](https://mosdef.org) in the [MoSDeF Cassandra docs](https://mosdef-cassandra.readthedocs.io/en/latest/) under "Resources" on the homepage.
